### PR TITLE
tidy up README

### PR DESCRIPTION
**Pull request by orangecms**

### CODE REVIEW

**Review by fotonick — COMMENTED** [review]
I personally like this tidying, but would like to get the approval from @rjoleary or someone who crafted the current README.

File: README.md [hunk 1]
```diff
@@ -11,17 +11,15 @@ oreboot is mostly written in Rust, with assembly where needed.
...
+-----
 
-Oreboot+QEMU for ARM:
+- [oreboot for ARM in QEMU](https://asciinema.org/a/Ne4Fwa4Wpt95dorEoVnHwiEkP)
+- [oreboot for RISC-V HiFive Unleased in QEMU](https://asciinema.org/a/XnWkMWTABuajsbGPMMTefjuZ2)
 
-[![asciinema](https://asciinema.org/a/Ne4Fwa4Wpt95dorEoVnHwiEkP.png)](https://asciinema.org/a/Ne4Fwa4Wpt95dorEoVnHwiEkP)
+Screenshot from RISC-V demo:
 
+![RISC-V QEMU](Documentation/img/qemu-riscv-demo.png)
```
Comment (rjoleary) [inline README.md:22]: Would it be possible to make this an SVG or copy in the text instead to avoid boating the git repo too much?

Comment (orangecms) [reply README.md:22]: Definitely, then I'll go for the latter. I just took a screenshot.

**Comment by fotonick** [comment]
Hi, is it convenient to resurrect this PR in fairly short order? I notice that the readme is now in a broken state with missing closing markup and an archived asciinema video.

**Comment by orangecms** [comment]
Yea, definitely. I'll get back to it now by the weekend. :)

**Comment by orangecms** [comment]
Done :) :tada: :sparkles:

**Review by fotonick — APPROVED** [review]
Very good. Thank you for fixing things!

I see that there area missing back-ticks to close a codeblock on line 161. Would you be willing to make one more commit to fix that? If not, I'll take care of it later.

**Comment by orangecms** [comment]
Ah, right, that one... fixed :)

**Review by fotonick — APPROVED** [review]
Looks awesome. Thanks so much for the interrupts and extra request. Happy holidays!

**Pull request merged**